doc/rst/language/spec/locales.rst

```diff
@@ -22,14 +23,15 @@ running on another locale.
 Locales
 -------
 
-A *locale* is a portion of the target parallel architecture that has
-processing and storage capabilities. Chapel implementations should
-typically define locales for a target architecture such that tasks
-running within a locale have roughly uniform access to values stored in
-the locale’s local memory and longer latencies for accessing the
-memories of other locales. As an example, a cluster of multicore nodes
-or SMPs would typically define each node to be a locale. In contrast a
-pure shared memory machine would be defined as a single locale.
+A *locale* is a Chapel abstraction for a piece of a target
+architecture that has processing and storage capabilities.
+Generally speaking, the tasks running within a locale have
+roughly uniform access to values stored in the locale's local
+memory and longer latencies for accessing the memories of other
+locales.  As examples, a single shared memory machine would be
+defined as a single locale, while in a system consisting of a
```

Comment:
I think it would probably be better to split this back into two sentences, but it is more clear to have the single locale  example first and I think the paragraph as a whole is an improvement

Reply:
OK, I've updated the wording. If you are able, make sure you are happy with the change.

Reply:
Yup, looks good!